library/src/lib/action-bar/action-bar-actions.component.ts

```diff
@@ -1,15 +1,20 @@
 import { Component, ElementRef, Inject } from '@angular/core';
 import { AbstractFdNgxClass } from '../utils/abstract-fd-ngx-class';
 
+/**
+ * This component holds the right-aligned action buttons for the action bar.
+ */
```

Comment:
Add example of usage inside the action bar since it's a very limited component

Reply:
Can do, I just thought that might be a little redundant since examples are on the examples tab